lib/open_mic.rb

```diff
@@ -0,0 +1,33 @@
+class OpenMic
+
+  attr_reader :location, :date, :performers
+
+  def initialize(event_info)
+    @event_info = event_info
```

Comment:
since we never ask for the whole hash of data bask, you don't need an instance variable for `@event_info`